app/scripts/lib/router.js

```diff
@@ -117,6 +117,13 @@ const Router = Backbone.Router.extend({
     'oauth/force_auth(/)': createViewHandler(ForceAuthView),
     'oauth/signin(/)': 'onSignIn',
     'oauth/signup(/)': 'onSignUp',
+    'pair/auth(/)': createViewHandler('pairing/pair_auth'),
+    'pair/auth/allow(/)': createViewHandler('pairing/pair_auth_allow'),
+    'pair/auth/complete(/)': createViewHandler('pairing/pair_auth_complete'),
+    'pair/supp(/)': createViewHandler('pairing/pair_supp'),
+    'pair/supp/allow(/)': createViewHandler('pairing/pair_supp_allow'),
+    'pair/supp/complete(/)': createViewHandler('pairing/pair_supp_complete'),
```

Comment:
I don't *think* this route is ever used, the OAuth flow redirects back to the app before showing this.